src/components/selectable/selectable_templates/selectable_template_sitewide.tsx

```diff
@@ -0,0 +1,221 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, {
+  FunctionComponent,
+  ReactNode,
+  useState,
+  CSSProperties,
+} from 'react';
+import classNames from 'classnames';
+import { useCombinedRefs } from '../../../services';
+import { EuiSelectable, EuiSelectableProps } from '../selectable';
+import { EuiPopoverTitle, EuiPopoverFooter } from '../../popover';
+import { EuiPopover, Props as PopoverProps } from '../../popover/popover';
+import { useEuiI18n, EuiI18n } from '../../i18n';
+import { EuiSelectableMessage } from '../selectable_message';
+import { EuiLoadingSpinner } from '../../loading';
+import {
+  EuiSelectableTemplateSitewideOption,
+  euiSelectableTemplateSitewideFormatOptions,
+  euiSelectableTemplateSitewideRenderOptions,
+} from './selectable_template_sitewide_option';
+
+export type EuiSelectableTemplateSitewideProps = Partial<EuiSelectableProps> & {
```

Comment:
```suggestion
export type EuiSelectableTemplateSitewideProps = Partial<
  Omit<EuiSelectableProps, 'options'>
> & {
```


Because it's redeclared later